packages/@internationalized/message/src/MessageDictionary.ts

```diff
@@ -26,7 +26,10 @@ export class MessageDictionary {
 
   constructor(messages: LocalizedStrings, defaultLocale: string = 'en-US') {
     // Clone messages so we don't modify the original object.
-    this.messages = {...messages};
+    // Filter out entries with falsy values which may have been caused by applying optimize-locales-plugin.
+    this.messages = Object.fromEntries(
+      Object.entries(messages).filter(([, v]) => v)
+    );
     this.defaultLocale = defaultLocale;
   }
```

Comment:
MessageDictionary is actually deprecated and unused, just here for backwards compatibility. We use LocalizedStringDictionary now (has much smaller bundle size).

Reply:
And I was wondering why we have almost same code in both places. That makes more sense now.